controller/test/blower_fsm/blower_fsm_test.cpp

```diff
@@ -69,6 +69,13 @@ void testSequence(
   }
 }
 
+constexpr int64_t rise_time_us = RISE_TIME.microseconds();
+static_assert(rise_time_us % 1000 == 0,
+              "blower fsm tests assume rise time is a whole number of ms.");
+constexpr int64_t rise_time_ms = rise_time_us / 1000;
+static_assert(rise_time_ms % 4 == 0,
+              "blower fsm tests assume we can divide rise time ms by 4.");
```

Comment:
Put these global variables at the top alongside inputs_zero?

Reply:
sure thing.